aspnetcore/includes/docker-128/Program.cs

```diff
@@ -0,0 +1,27 @@
+using Docker128;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+
+#region snippet
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        CreateHostBuilder(args).Build().Run();
+    }
+
+public static IHostBuilder CreateHostBuilder(string[] args) =>
+    Host.CreateDefaultBuilder(args)
+        .ConfigureAppConfiguration((hostingContext, config) =>
+        {
+            config.AddJsonFile("appsettings.json",
```

Comment:
This doesn't fix the issue because CreateDefaultBuilder already called `config.AddJsonFile("appsettings.json", optional: true, reloadOnChange: true);`, calling it a second time doesn't turn off the prior calls reload notifications. DOTNET_HOSTBUILDER__RELOADCONFIGONCHANGE=false is the proper fix.